src/styles/blocks/button.scss

```diff
@@ -0,0 +1,55 @@
+.button {
+  color: #f1f5f4;
+  background-color: #cd4d31;
```

Comment:
It is better to create variables for your colors and main values like transition time etc in separate _variables.scss file